articles/provider-1qbit.md

```diff
@@ -86,21 +79,20 @@ Please note that all parameter values are required to be of JSON string type.  T
 
 The path-relinking algorithm is a heuristic algorithm that uses the tabu search as a subroutine to solve a QUBO problem. The algorithm starts from a set of elite solutions found by the tabu search. It then constructs a path between each pair of elite solutions, selects one of the solutions along the path, and repeats the tabu search. If the tabu search finds a distinct solution that is better than the current worst elite solution, the elite solutions set is updated with the new improved solution. This whole procedure is repeated until the algorithm meets a stopping condition.
 
+>[NOTE!]
+> If not using the Python SDK, parameter values are required to be of JSON string type. The types listed here are the types that the solvers expect within the string value. For example, `greedy_path_relinking` is listed as type boolean, and is expected to be passed in the format:
+>
+>**greedy_path_relinking: "False"**
+
 - Job type: `Quantum-Inspired Optimization Problem`
 - Data Format: `microsoft.qio.v2`
 - Target ID: `1qbit.pathrelinking`
 - Python Solver class name: `PathRelinkingSolver`
 
-******
-Please note that all parameter values are required to be of JSON string type.  The types listed below are the types the solvers expect within the string value.  For example, 'greedy_path_relinking' is listed as type boolean, so it is expected to be given in the format:
-
-"greedy_path_relinking": "false"
-******
-
 | Parameter Name            | Type     | Required | Description |
 |---------------------------|----------|----------|-------------|
 
-| distance_scale|double|Optional|The minimum distance from the initiating and guiding solutions for constructing the candidate solution list. The highest-quality solution in the candidate solution list is then selected for improvement. Valid values: 0.0 to 0.5. Default value: 0.33. | 
-| greedy_path_relinking|boolean|Optional|When you use the path-relinking solver, there are two ways you can generate a path that leads to the solution: one is the greedy function and the other operates in a random matter. If you set this parameter to **True**, the solver will use the greedy function. If you set it to **False**, it will use the random method. Default value: false |
+| distance_scale|double|Optional|The minimum distance from the initiating and guiding solutions for constructing the candidate solution list. The highest-quality solution in the candidate solution list is then selected for improvement. Valid values: 0.0 to 0.5. Default value: 0.33. |
+| greedy_path_relinking|boolean|Optional|When you use the path-relinking solver, there are two ways you can generate a path that leads to the solution: one is the greedy function and the other operates in a random matter. If you set this parameter to **True**, the solver will use the greedy function. If you set it to **False**, it will use the random method. Default value: **False** |
```

Comment:
```suggestion
| greedy_path_relinking|boolean|Optional|When you use the path-relinking solver, there are two ways you can generate a path that leads to the solution: one is the greedy function and the other operates in a random matter. If you set this parameter to **True**, the solver will use the greedy function. If you set it to **False**, it will use the random method. Default value: False |
```

Reply:
This should be non-bold to match the other Default value formatting.

Reply:
Perfect, thank you!